### ISSUE TITLE
Overly aggressive automatic indent on deeply nested list in a `@pytest.mark.parametrize()` call

### ISSUE THREAD

**Issue by brettcannon** [microsoft/pylance-release#3839]
<h3> Environment data </h3>

-   Language Server version: 2023.1.21
-   OS and version: win32 x64
-   Python version (and distribution if applicable, e.g. Anaconda): 
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: off

<h3> Code Snippet </h3>

The blank line in the middle of the `pytest.mark.parametrize()` decorator gets indented 11 time too far to the right (inline with the `[` on the previous line instead of with `(`).

```python
import pytest


class TestStuff:
    @pytest.mark.parametrize(
            ["given", "expected"],
            [
                ("A", ["A"]),
                ("A ", ["A"]),
                (" A", ["A"]),
                ("A, B", ["A", "B"]),
                ("A,B", ["A", "B"]),
                (" A, B", ["A", "B"]),
                           
                ("A B", ["A B"]),
            ],
        )
    def test(given, expected):
        pass
```

<h3> Repro Steps </h3>

1. XXX

<h3> Expected behavior </h3>

Indent should align with the `(` as the overall list is still "open" while everything on the previous line in the tuple is "closed".

<h3> Actual behavior </h3>

Indents like the list on the previous line is still being added to.

<h3> Logs </h3>

```
[Info  - 8:56:39 AM] (22680) Pylance language server 2023.1.21 (pyright bf8ec313) starting
[Info  - 8:56:39 AM] (22680) Server root directory: c:\Users\brcan\.vscode-insiders\extensions\ms-python.vscode-pylance-2023.1.21\dist
[Info  - 8:56:39 AM] (22680) Starting service instance "Some Python scratch space"
[Info  - 8:56:39 AM] (22680) Auto-indent enabled
[Info  - 8:56:39 AM] (22680) No configuration file found.
[Info  - 8:56:39 AM] (22680) pyproject.toml file found at c:\Users\brcan\Testing\Some Python scratch space.
[Info  - 8:56:39 AM] (22680) Setting pythonPath for service "Some Python scratch space": "c:\Users\brcan\Testing\Some Python scratch space\.venv\Scripts\python.exe"
[Info  - 8:56:39 AM] (22680) Loading pyproject.toml file at c:\Users\brcan\Testing\Some Python scratch space\pyproject.toml
[Error - 8:56:39 AM] (22680) Pyproject file "c:\Users\brcan\Testing\Some Python scratch space\pyproject.toml" is missing "[tool.pyright]" section.
[Warn  - 8:56:39 AM] (22680) stubPath c:\Users\brcan\Testing\Some Python scratch space\typings is not a valid directory.
[Info  - 8:56:39 AM] (22680) Assuming Python version 3.11
[Info  - 8:56:39 AM] (22680) Assuming Python platform Windows
[Info  - 8:56:40 AM] (22680) Searching for source files
[Info  - 8:56:40 AM] (22680) Found 5 source files
[Info  - 8:56:40 AM] (22680) Background analysis(1) root directory: c:\Users\brcan\.vscode-insiders\extensions\ms-python.vscode-pylance-2023.1.21\dist
[Info  - 8:56:40 AM] (22680) Background analysis(1) started
[Info  - 8:56:40 AM] (22680) Indexer background runner(2) root directory: c:\Users\brcan\.vscode-insiders\extensions\ms-python.vscode-pylance-2023.1.21\dist (index)
[Info  - 8:56:40 AM] (22680) Indexing(2) started
[Info  - 8:56:42 AM] (22680) scanned(2) 42 files over 1 exec env
[Info  - 8:56:43 AM] (22680) indexed(2) 26 files over 1 exec env
[Info  - 8:56:43 AM] (22680) Indexing finished(2).
[Info  - 8:57:11 AM] (22680) Indexer background runner(3) root directory: c:\Users\brcan\.vscode-insiders\extensions\ms-python.vscode-pylance-2023.1.21\dist (refresh)
[Info  - 8:57:11 AM] (22680) Indexing(3) started
[Info  - 8:57:12 AM] (22680) scanned(3) 47 files over 1 exec env
[Info  - 8:57:13 AM] (22680) indexed(3) 31 files over 1 exec env
[Info  - 8:57:13 AM] (22680) Indexing finished(3).

```

**Comment by rchiodo** [comment]
Thanks. Looks like it's not finding the close bracket before the open bracket.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.1.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023131-18-January-2023-PreRelease)
